spec/system/admin/settings_spec.rb

```diff
@@ -340,4 +341,30 @@
       Setting["feature.user.skip_verification"] = nil
     end
   end
+
+  describe "SDG configuration tab" do
+    scenario "Should be able when sdg feature activated" do
```

Comment:
How about: "is enabled when the sdg feature is enabled"?

Reply:
Again This was also consistent with the rest of the file 😅 
I change it, I like the suggestion